### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  @Override
+  public boolean isDuplicate(String consumerId, String messageId) {
+    return false;
+  }
+
+  @Override
+  public void doWithMessage(SubscriberIdAndMessage subscriberIdAndMessage, Runnable callback) {
+    eventuateMicronautTransactionManagement.doWithTransaction(connection -> callback.run());
```
Comment (cer) [inline eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java]: This isn't right. doWithMessage() should simply call callback.run()

File: eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package io.eventuate.tram.consumer.jdbc.micronaut;
+
+import io.eventuate.common.jdbc.micronaut.EventuateMicronautTransactionManagement;
+import io.eventuate.tram.consumer.common.DuplicateMessageDetector;
+import io.eventuate.tram.consumer.common.SubscriberIdAndMessage;
+
+public class EventuateMicronautTransactionalNoopDuplicateMessageDetector implements DuplicateMessageDetector {
```
Comment (cer) [inline eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java]: Look at the original version. It has nothing to do with Spring. And, this should have nothing to do with Micronaut. See comment below. The same version should work for both.

Comment (dartartem) [reply eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java]: Chris, if this: https://github.com/dartartem/eventuate-tram-core/blob/master/eventuate-tram-consumer-jdbc/src/main/java/io/eventuate/tram/consumer/jdbc/TransactionalNoopDuplicateMessageDetector.java is original version, it uses spring transaction template. I do not understand.

File: eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java (hunk 1 above)
Comment (dartartem) [reply eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java]: But original version uses transactions

**Pull request merged**